core/src/main/java/com/avast/bytes/jdk/FileBackedBytes.java

```diff
@@ -0,0 +1,93 @@
+package com.avast.bytes.jdk;
+
+import com.avast.bytes.AbstractBytes;
+import com.avast.bytes.Bytes;
+import com.avast.bytes.internal.ByteBufferInputStream;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+/**
+ * Implementation of {@link Bytes} backed by memory mapped file  in format of {@link ByteBuffer}.
```

Comment:
I would remove the note about `ByteBuffer`. That's an implementation detail that's quite obvious from the fact that you are doing memory-mapped files.